file_upload_handler/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets":["@babel/preset-env"]
```

Comment:
How come you decided to use Babel?
Babel is for making Javascript compatible with older browsers/environments, but this is gonna run on a server so that shouldn't be an issue?